docs/src/main/sphinx/release/release-432.md

```diff
@@ -0,0 +1,57 @@
+# Release 432 (2 Nov 2023)
+
+:::{note}
+From this point forward, release notes will include a ⚠️ symbol to designate
+breaking changes. Breaking changes may:
+* Require users to update their configuration, such as if a property is removed
+* Update the default values for configuration properties, which may change
+  behavior
+* Require external systems or software to be updated alongside Trino, such as
```

Comment:
Update of requirements for external systems or software used with Trino, such as removal support for an old version of a data source in a connector.